docs/general/oasis-network/why-oasis.md

```diff
@@ -161,16 +161,14 @@ If you want to build on Oasis, you can read more about the Ecosystem Fund [here]
 
 The Oasis Foundation has been working with many talented dev teams via our [Grants program](https://oasisprotocol.org/ecosystem#grantsprogram) to build new applications and integrations on top of the Oasis Network. These include:
 
-* [Anthem (by Chorus One)](https://anthem.chorus.one)
 * [Alethea.ai](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-alethea-ai-aac8f854e436): A marketplace for AI-generated media
 * [Bankex](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-bankex-9bb127f9e449): Send digital assets across your social networks
 * [Castalise](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-castalise-347f155ab66f): Enabling privacy-preserving predictive analytics on pharmaceutical data
 * [Chainflow Mission Control](https://github.com/Chainflow/oasis-mission-control-grant)
 * [Dead Man’s Switch](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-dead-mans-switch-32d07cdfc057): A decentralized, censorship-resistant tool for whistleblowers
 * [Luther.ai:](https://luther.ai)  Building artificial intelligence to Retain, Reinforce, Recall and ultimately augment your human memory.
 * [Oasis.Fish (by Stakefish)](https://oasis.fish/leaderboard/)
-* [Oasis Hubble (by Figment Networks)](https://www.figment.io/)
-* [Oasis Monitor (by Everstake)](https://oasismonitor.com)
```

Comment:
Also noticed this one was deprecated while clicking around. cc @matevz

Reply:
Would be useful, if we searched through the docs for any Oasis Monitor mentions and remove the links.

Reply:
Let's do that with https://github.com/oasisprotocol/docs/issues/987